io.github.iptux_src.iptux.yml

```diff
@@ -0,0 +1,58 @@
+# snapcraft.yaml
+name: iptux
+version: 0.9.3
```

Comment:
not sure what documentation you have been reading, but this doesn't look like things found in flatpak manifest.